pkgs/build-support/setup-hooks/add-bin-to-path.sh

```diff
@@ -0,0 +1,15 @@
+# shellcheck shell=bash
+# This setup hook add $out/bin to the PATH environment variable.
+
+export PATH
+
+addBinToPath () {
+    # shellcheck disable=SC2154
+    if [ -d "$out/bin" ]; then
```

Comment:
Shouldn't we log something if the directory doesn't exist?

Reply:
It hasn't been suggested during the month this PR has been opened, so no.
That said, while implementing that hook this morning, I had an issue because of that if condition. I removed it in https://github.com/NixOS/nixpkgs/pull/378110/commits/62d4ca6101ac9644331c4d7edfb408569c6ed86a